src/models/tiles/text/text-content.test.ts

```diff
@@ -81,38 +74,17 @@ describe("TextContentModel", () => {
     expect(bogus2.asSlate()).toBeDefined();
   });
 
-  const fooJson: ValueJSON = {
-                document: {
-                  nodes: [{
-                    object: "block",
-                    type: "paragraph",
-                    nodes: [{
-                      object: "text",
-                      text: "foo"
-                    }]
-                  }]
-                }
-              };
+  const value: EditorValue = [
+    {
+      type: 'paragraph',
+      children: [{ text: 'foo' }],
+    },
+  ];
 
   it("converts to slate correctly", () => {
     const foo = "foo";
     const model = TextContentModel.create({ text: foo });
-    expect(Plain.serialize(model.asSlate())).toBe(foo);
-
-    jestSpyConsole("warn", mockConsole => {
```

Comment:
I removed these tests since markdown isn't implemented yet, but maybe they should stay and be skipped?